launcher_scripts/conf/cluster/k8s_v2.yaml

```diff
@@ -21,5 +21,9 @@ ib_interfaces:
     # nvidia.com/hostdev: 8
 
 dns_policy: null  # Specify a dnsPolicy to use in all pods, if necessary
-pull_secret: ngc-registry  # Kubernetes secret for the container registry to pull private containers.
-capabilities: []
\ No newline at end of file
+pull_secret: ngc-registry  # Kubernetes secret for the container registry to pull private containers. Set to "null" if the imagePullSecret is managed by the scheduler or otherwise not needed.
+custom_mpijob_api_version: null  # Optionally specify a custom k8s API version for MPIJobs. Default is "kubeflow.org/v1". This occurs when a cluster has different API versions installed.
+custom_pytorchjob_api_version: null  # Optionally specify a custom k8s API version for PyTorchJobs. Default is "kubeflow.org/v1". This occurs when a cluster has different API versions installed.
```

Comment:
I thought v1 was the API version for a while now, how old of an operator are we dealing with? If it's super old, my gut reaction is to just do `pip install kubeflow-training==$some_other_ver`

Granted the python sdk code could look very different so I'm not sure this code will run with the packages/modules the way they're configured, so other changes might be involved if we go that route

Reply:
Also, I see you have separate versions for mpijob and pytorchjob, does that mean this is before the training-operator consolidated the operators? I would prefer no extra flag, but one flag is preferable to two.